src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoWUser } from '../../types/TodoWUser';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
```

Comment:
```suggestion
interface  Props = {
```
use interface prefer